src/main/java/uk/gov/mca/beacons/service/accounts/AccountsService.java

```diff
@@ -0,0 +1,21 @@
+package uk.gov.mca.beacons.service.accounts;
+
+import javax.transaction.Transactional;
+import org.springframework.stereotype.Service;
+import uk.gov.mca.beacons.service.model.AccountHolder;
+import uk.gov.mca.beacons.service.repository.AccountHolderRepository;
+
+@Service
+@Transactional
+public class AccountsService {
```

Comment:
If we're following the single responsibility principle and having use cases performing one interaction a user has with the service it might be better to rename this service to `AccountsGetByAuthIdService` or something similar?

Reply:
Agreed!  Renamed to `GetAccountHolderByAuthIdService`